libs/coin-framework/src/api/types.ts

```diff
@@ -60,7 +60,10 @@ export type Api<TokenIdentifier> = {
   broadcast: (tx: string) => Promise<string>;
   combine: (tx: string, signature: string, pubkey?: string) => string;
   estimateFees: (transactionIntent: TransactionIntent<TokenIdentifier>) => Promise<bigint>;
-  craftTransaction: (transactionIntent: TransactionIntent<TokenIdentifier>) => Promise<string>;
+  craftTransaction: (
+    transactionIntent: TransactionIntent<TokenIdentifier>,
+    customFees?: bigint,
```

Comment:
@jnicoulaud-ledger lama-adapter takes an arbitrary Json that is forwarded 
to alpaca that must decapsulate the nested-nested value
see https://github.com/LedgerHQ/lama-adapter/blob/e87543566df5e34b73b8c4d8f9b3015f37d6aef2/modules/api/src/main/scala/lama/api/requests/EncodeRequest.scala#L10
see https://github.com/LedgerHQ/alpaca-coin-module/blob/272e7419b000e72c2f22861d71967847b75671be/src/routes/encode.ts#L87

I suggest we use BigInt in lama-adapter and simplify the alpaca api.
I don't understand we choosed a "whatever" type here in our codebase.

Reply:
ok for me